### PR TITLE
Version EN

### PR DESCRIPTION
**Pull request by croustibat**
Add routing and file to create english version

### CODE REVIEW

**Comment by vvo** [comment]
Having a look now

**Comment by vvo** [comment]
@croustibat can you run:

```shell
yarn format
```

so that it will reformat files automatically

File: components/Menu.jsx [hunk 1]
```diff
@@ -91,14 +108,12 @@ function Nav({ allDates, toggleSidebar }) {
...
       })}
-
       <HandDrawnLine className="text-yellow-700 my-4 h-2 w-32 opacity-50 mx-auto" />
-
       <a
         href="https://simpleanalytics.com/coronamaison.net"
         className="block text-xl text-blue-800 font-cursive hover:text-twitter"
       >
-        Statistiques
+        {lang === "en" ? `Statistics` : `Statistiques`}
```
Comment (vvo) [inline components/Menu.jsx:116]:
```suggestion
        {lang === "en" ? `Analytics` : `Statistiques`}
```

File: pages/en/i-want-to-draw.jsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        </a>
+        , here is how easy it is:
+      </p>
+
+      <ol className="mt-4">
+        <li className="text-lg">
+          1.{" "}
+          <a
+            href="/modele-coronamaison.jpg"
+            alt="Modèle de dessin Coronamaison"
```
Comment (vvo) [inline pages/en/i-want-to-draw.jsx:47]:
```suggestion
            alt="Coronamaison drawing template"
```

File: pages/en/index.jsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Head from "next/head";
+import Link from "next/link";
+
+import DrawingsList from "components/DrawingsList";
+import Share from "components/Share";
+
+export default function Home({ top50Drawings }) {
+  // const title = "CoronaMaison : Tous les dessins du challenge artistique";
```
Comment (vvo) [inline pages/en/index.jsx:10]:
```suggestion
```

File: pages/en/index.jsx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+  const title = "CoronaMaison : All the drawings of the artistic challenge";
+  // const description = "Ce site contient tous les dessins du hashtag #CoronaMaison publiés sur les réseaux sociaux. Ils sont organisés par date et popularité. Découvre-les !";
```
Comment (vvo) [inline pages/en/index.jsx:12]:
```suggestion
```

**Comment by vvo** [comment]
Deployment is failing because ... We hit the zeit limit on output files since we duplicated every drawing/id/en AH!

Two solutions:
1. we merge this but we comment for now the /en/drawing/id pages to always use the fr one so we can merge it and it works! Then when we implement image upload to OVH, we can put it back
2. we implement static generation with a fallback so we do not generate every drawing page statically (https://nextjs.org/docs/basic-features/data-fetching#the-fallback-key-required)

Let me know which path you want to go

**Comment by vvo** [comment]
Just made the deployments of coronamaison public but might take some time to work. Still here's the screenshot of the error:

File: components/Menu.jsx (hunk 1 above)
Comment (croustibat) [reply components/Menu.jsx:116]: done

**Comment by croustibat** [comment]
Thank for reviewing. 
I committed some fixes.

For the deployment, the first solution is the best ! we merge this PR then comment until the new media hosting strategy is working.

**Comment by vvo** [comment]
> For the deployment, the first solution is the best ! we merge this PR then comment until the new media hosting strategy is working.

Is it possible to solve this before we merge then? So that we merge something that is working and we can check the preview online, let me know!

**Comment by vvo** [comment]
You also still have some linting/formatting issues. You can try `yarn format` and `yarn lint` and then fix manually

**Pull request closed without merging**